### PR TITLE
Front end fixes

### PR DESCRIPTION
**Pull request by SaraCah**
Added the feedback form dropdown to get it working with the bootstrap react conflict and changed the placeholder text with the search bar to say search all of qld.gov.au

### CODE REVIEW

File: config/config.rb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # create database qsa_public character set UTF8mb4 collate utf8mb4_bin;
 # grant all on qsa_public.* to 'qsa'@'localhost' identified by 'qsa123';
 
-# AppConfig[:db_url] = "jdbc:mysql://localhost:3306/qsa_public?useUnicode=true&characterEncoding=UTF-8&user=qsa&password=qsa123&serverTimezone=UTC"
-
+AppConfig[:db_url] = "jdbc:mysql://localhost:3306/qsa_public?useUnicode=true&characterEncoding=UTF-8&user=qsa&password=qsa123"
```
Comment (marktriggs) [inline config/config.rb]: I think we probably don't need to commit changes to `config.rb` here.  But, a tip for running a dev environment: if you put this line into a file called `config.local.rb` alongside this one, it'll get loaded in the same way.  `config.local.rb` is in `.gitignore` so it doesn't get committed by mistake.

File: package-lock.json [hunk 2]
```diff
@@ -1,3 +1,1303 @@
 {
```
Comment (marktriggs) [inline package-lock.json]: I think this file isn't actually used.  It kind of looks like I committed it by mistake last year :)  Would you mind `git rm`'ing that file and amending this PR to drop it out?

**Comment by SaraCah** [comment]
Hi @marktriggs ! Think I resolved what was up.

**Comment by marktriggs** [comment]
Perfect!  Thanks again :)  :pineapple:

**Pull request merged**